web/js/components/measure-tool/measure-menu.js

```diff
@@ -0,0 +1,146 @@
+import PropTypes from 'prop-types';
+import { connect } from 'react-redux';
+import React, { Component } from 'react';
+import { onToggle } from '../../modules/modal/actions';
+import IconList from '../util/list';
+import { changeUnits, useGreatCircle } from '../../modules/measure/actions';
+import { Form, FormGroup, Label, Input, Tooltip } from 'reactstrap';
+// import googleTagManager from 'googleTagManager';
```

Comment:
I would highly recommend getting analytics in place before launch. @edplato should be able to help you set it up on the GTM side.

Reply:
Sounds good; I talked with Ed about this a little last week.  Will follow up and get with him next week so we can have this included for the release.